src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -135,89 +114,22 @@ public function testWrite()
             ->with($this->options['database'], $this->options['collection'])
             ->will($this->returnValue($collection));
 
-        $data = array();
-
-        $methodName = phpversion('mongodb') ? 'updateOne' : 'update';
-
         $collection->expects($this->once())
-            ->method($methodName)
-            ->will($this->returnCallback(function ($criteria, $updateData, $options) use (&$data) {
+            ->method('updateOne')
+            ->will($this->returnCallback(function ($criteria, $updateData, $options) {
                 $this->assertEquals(array($this->options['id_field'] => 'foo'), $criteria);
-
-                if (phpversion('mongodb')) {
-                    $this->assertEquals(array('upsert' => true), $options);
-                } else {
-                    $this->assertEquals(array('upsert' => true, 'multiple' => false), $options);
-                }
+                $this->assertEquals(array('upsert' => true), $options);
 
                 $data = $updateData['$set'];
+                $expectedExpiry = time() + (int) ini_get('session.gc_maxlifetime');
+                $this->assertInstanceOf(\MongoDB\BSON\Binary::class, $data[$this->options['data_field']]);
+                $this->assertEquals('bar', $data[$this->options['data_field']]->getData());
+                $this->assertInstanceOf(\MongoDB\BSON\UTCDateTime::class, $data[$this->options['time_field']]);
+                $this->assertInstanceOf(\MongoDB\BSON\UTCDateTime::class, $data[$this->options['expiry_field']]);
+                $this->assertGreaterThanOrEqual($expectedExpiry, round((string) $data[$this->options['expiry_field']] / 1000));
             }));
 
-        $expectedExpiry = time() + (int) ini_get('session.gc_maxlifetime');
         $this->assertTrue($this->storage->write('foo', 'bar'));
-
-        if (phpversion('mongodb')) {
-            $this->assertEquals('bar', $data[$this->options['data_field']]->getData());
-            $this->assertInstanceOf('MongoDB\BSON\UTCDateTime', $data[$this->options['time_field']]);
-            $this->assertInstanceOf('MongoDB\BSON\UTCDateTime', $data[$this->options['expiry_field']]);
-            $this->assertGreaterThanOrEqual($expectedExpiry, round((string) $data[$this->options['expiry_field']] / 1000));
-        } else {
-            $this->assertEquals('bar', $data[$this->options['data_field']]->bin);
-            $this->assertInstanceOf('MongoDate', $data[$this->options['time_field']]);
-            $this->assertInstanceOf('MongoDate', $data[$this->options['expiry_field']]);
-            $this->assertGreaterThanOrEqual($expectedExpiry, $data[$this->options['expiry_field']]->sec);
-        }
-    }
-
-    public function testWriteWhenUsingExpiresField()
```

Comment:
this test doing the same as the one above. sometime in the past the expiry_field option seemed to be optional.